api/v1alpha08/trait/base.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2023 Red Hat, Inc. and/or its affiliates
```

Comment:
instead being inside v1alpha08, maybe move to another place, maybe api/common_objects.go or something else, wdyt?

Reply:
@spolti As I am not using this struct anymore, this file base.go became unnecessary. Also I moved out the trait directory under api/trait to simplify the hierarchy. Let me know what you think.